domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraUseCase.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.jetpackcamera.domain.camera
+
+import androidx.camera.core.CameraSelector
+import androidx.camera.core.Preview
+import androidx.lifecycle.LifecycleOwner
+
+/**
+ * Data layer for camera.
+ */
+interface CameraUseCase {
+
+    /**
+     * Initializes the camera.
+     */
+    suspend fun initialize(): List<Int>
+
+    /**
+     * Starts the camera preview.
```

Comment:
Same here -- for return type
I'm assuming true if the preview started and false otherwise?
But is strting preview async?